cmd/natc/natc.go

```diff
@@ -281,17 +356,20 @@ func (c *connector) generateDNSResponse(req *dnsmessage.Message, from tailcfg.No
 			}
 			found = true
 			if want6 {
-				err = b.AAAAResource(
+				if err := b.AAAAResource(
```

Comment:
don't intermix error checking patterns. Either update all of them, or let the return below catch all